math/median_test.go

```diff
@@ -0,0 +1,59 @@
+// author(s) [jo3zeph](https://github.com/jo3zeph)
+// median_test.go
+// see median.go
+
+package math
```

Comment:
```suggestion
package math_test
```

Reply:
There isn't a package called math_test correct? May i know the reason behind this change?

Reply:
This suggestion has caused another error in my `median_test.go`, may I know how to fix it?
![image_2022-08-07_121334046](https://user-images.githubusercontent.com/86296674/183274990-18b5a1b1-e8e5-4123-a2cb-69327da19e0e.png)